### PR TITLE
Lint update

### PR DESCRIPTION
**Pull request by pderouen**
Resolves https://github.com/Origen-SDK/origen/issues/440

### CODE REVIEW

File: lib/origen/application/lsf_manager.rb [hunk 1]
```diff
@@ -209,7 +209,7 @@ def clear(options)
           if options[:type] == :all
             File.delete(remote_jobs_file) if File.exist?(remote_jobs_file)
             @remote_jobs = {}
-            return
```
Comment (pderouen) [inline lib/origen/application/lsf_manager.rb:212]: This diff is caused by [this](https://github.com/Origen-SDK/origen/blob/d1143f82d2019202b06fa3bec2e6a5b0dc102878/config/rubocop/strict.yml#L4594) rule. Rubocop doesn't indicate that the rule has been updated.

We have this rule [enabled](https://github.com/Origen-SDK/origen/blob/e531b9c03573a6417a4588ca520bcafa8aa41eb8/config/rubocop/strict_enabled.yml#L392-L395) in our existing rules.

Most of the diffs have a similar story.

**Review by coreyeng — APPROVED** [review]
Looks good and thanks for doing this! We'll have to bite the bullet and update the linter at some point so probably sooner rather than later is better anyway.

**Comment by pderouen** [comment]
> Looks good and thanks for doing this! We'll have to bite the bullet and update the linter at some point so probably sooner rather than later is better anyway.

Thanks. I'll give @priyavadan and @info-rchitect  a few more days to chime in if they'd like any changes.

**Review by info-rchitect — APPROVED** [review]
LGTM, agreed that we need to bite the bullet

**Pull request merged**